### DISCUSSION TITLE
检测检测模型微调PP-OCRv3和v4

### DISCUSSION THREAD

**Discussion by cloudfool** [PaddlePaddle/PaddleOCR#12776] [Q&A]
我看文档中写道：建议选择PP-OCRv3模型（配置文件：[ch_PP-OCRv3_det_student.yml](https://github.com/PaddlePaddle/PaddleOCR/blob/main/configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_student.yml)，预训练模型：[ch_PP-OCRv3_det_distill_train.tar](https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_det_distill_train.tar)）进行微调，其精度与泛化性能是目前提供的最优预训练模型。
意思是PP-OCRv3比v4更适合微调吗？

**Comment by SWHL** [comment]
哪个文档，没有这意思吧

**Reply by cloudfool** [reply]
https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/finetune.md
2.2章节

**Reply by SWHL** [reply]
这文档估计是v4还没出的时候写的，建议用最新的哈
文档大部分都年久失修了
